### ISSUE TITLE
is you mailman module alive?

### ISSUE THREAD

**Issue by sandwitch** [nicwaller/puppet-mailman#37]
Hi Nic,

If you don't mind I would like to ask if your module is alive. We used is for a year or so however we are getting to much dependencies problems. Is it okay if I create some pull requests?

regards from Holland

Mark

**Comment by nicwaller** [comment]
@sandwitch No, I am no longer maintaining this module, nor am I reviewing pull requests.

You are welcome to start your own fork.

**Comment by sandwitch** [comment]
Thans. For your answer and the work. We willen look inro it.
Regards
